controller/control.go

```diff
@@ -741,56 +737,29 @@ func (c *Controller) Start(addresses ...string) error {
 			return err
 		}
 	}
-
-	revisionCounters := make(map[string]int64)
-	for _, r := range c.replicas {
-		counter, err := c.backend.GetRevisionCounter(r.Address)
-		if err != nil {
-			logrus.Errorf("GetRevisionCounter failed %s %v", r.Address, err)
-			return err
-		}
-		if counter > expectedRevision {
-			expectedRevision = counter
-		}
-		revisionCounters[r.Address] = counter
-	}
-
-	for address, counter := range revisionCounters {
-		if counter != expectedRevision {
-			logrus.Errorf("Revision conflict detected! Expect %v, got %v in replica %v. Mark as ERR",
-				expectedRevision, counter, address)
-			c.setReplicaModeNoLock(address, types.ERR)
-		}
-	}
-	for regrep := range c.RegisteredReplicas {
-		sendSignal = 1
-		for _, tmprep := range c.replicas {
-			if tmprep.Address == "tcp://"+regrep+":9502" {
-				sendSignal = 0
-				break
-			}
-		}
-		if sendSignal == 1 {
-			logrus.Infof("sending add signal to %v", regrep)
-			c.factory.SignalToAdd(regrep, "add")
-		}
-	}
-	for regrep := range c.RegisteredQuorumReplicas {
-		sendSignal = 1
-		for _, tmprep := range c.quorumReplicas {
-			if tmprep.Address == "tcp://"+regrep+":9502" {
-				sendSignal = 0
-				break
-			}
-		}
-		if sendSignal == 1 {
-			logrus.Infof("sending add signal to quorum %v", regrep)
-			c.factory.SignalToAdd(regrep, "add")
-		}
-	}
-	logrus.Info("Update volume status")
-	c.UpdateVolStatus()
-
+	/*
+	   * This code has been commented since we start with only one replica currently
```

Comment:
This commented code can be removed.. and, add a comment that this is moved to monitorPreload function